SERVER/targets.d/CONFIG.skel.py

```diff
@@ -0,0 +1,9 @@
+"""
+Configuration of syllabus server.
+Edit to fit development or deployment environment.
+
+"""
+
+PORT=8080
+DEBUG = True  # Set to False for production use
+secret_key = "fmavKUW0Mwmc0uVLJPx84OLEpS6taGMBS5e1LwZ5lKUOItRZvFaVPTW0B6rEJVhJYdPNPs7Npw5gH0IG"
```

Comment:
Revealed a couple of dead files and comments that weren't correct anymore in the autoconfig. Tweaking them to match.